lib/db.js

```diff
@@ -289,17 +289,24 @@ class LimitDBRedis extends EventEmitter {
   }
 
   takeElevated(params, callback) {
-    const valError = validateERLParams(params.elevated_limits);
-    if (valError) {
-      return callback(valError)
+    const valERLParamsError = validateERLParams(params.elevated_limits);
+    if (valERLParamsError) {
+      return callback(valERLParamsError)
     }
     const erlParams = utils.getERLKeysQuotaAmountAndExpiration(params.elevated_limits);
 
     this._doTake(params, callback, (key, bucketKeyConfig, count) => {
-      const valError = validateConfigIsForElevatedBucket(key, bucketKeyConfig)
-      if (valError) {
-        return callback(valError)
+      let isERLEnabledForBucket = true;
+      const valERLConfigError = validateConfigIsForElevatedBucket(key, bucketKeyConfig)
+      if (valERLConfigError) {
+        isERLEnabledForBucket = false
```

Comment:
```suggestion
      const valERLConfigError = validateConfigIsForElevatedBucket(key, bucketKeyConfig)
      const isERLEnabledForBucket = !valERLConfigError;
      if (valERLConfigError) {
      
 ```
 
 to reduce code with mutable variables

Reply:
I originally wrote it like that, but I thought it would be clearer for the reader to understand that we're disabling ERL after not finding ERLConfig.
Do you find it clear what the intention is this way `const isERLEnabledForBucket = !valERLConfigError `?
If you think it is, I'll apply the change right away!

Reply:
I think it's better. I see what you mean; I think both ways are sufficiently clear, but not having to follow around mutable variables is much easier to follow imo